src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageAccountProvider.cs

```diff
@@ -153,6 +153,7 @@ private IStorageAccount StorageAccount
         public async Task<IStorageAccount> GetAccountAsync(string connectionStringName, CancellationToken cancellationToken)
         {
             IStorageAccount account = null;
+            var isPrimary = true;
```

Comment:
avoiding the StorageAccount / DashboardAccount getters and setters as they invoke parsing